plugins/templates/removeVersion.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const boom = require('@hapi/boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const nameSchema = schema.models.template.base.extract('name');
+const versionSchema = schema.models.template.base.extract('version');
```

Comment:
just thought of something
would it be better to use `exactVersion` schema? https://github.com/screwdriver-cd/data-schema/blob/master/config/template.js#L80

what happens if a user puts `12.0` vs `12.0.0` if we use `version` schema?

Reply:
We look for exact match in DB while fetching the template. 

When version `12.0` is specified, 404 is returned with error message saying template with version `12.0` does not exist. Version `12.0.0` is not removed when `12.0` is specified in the request.
**Sample Error**
```javascript
{
    "statusCode": 404,
    "error": "Not Found",
    "message": "Template sagar1312/sd-template-parameters with version 12.0 does not exist"
}

```


With `exactVersion`, it is fail-fast with a generic error
```
{
    "statusCode": 400,
    "error": "Bad Request",
    "message": "Invalid request params input"
}
```

Made changes to use `exactVersion` while validating the input.